src/databuffer.ts

```diff
@@ -45,16 +45,20 @@ export class DataBuffer {
 
         if (this.byteLength >= n) {
             this.pack();
-            let v = this.buffers.pop();
-            if (v) {
-                let d = v.slice(0, n);
+            const dataBuffer = this.buffers.pop();
```

Comment:
This is a local variable that spans very few lines. The new name makes it seem as an important variable or object, please keep the previous name.